src/Linfo/OS/Windows.php

```diff
@@ -553,10 +553,13 @@ public function getNet()
                     break;
             }
             // @Microsoft: An index would be nice here indeed.
-            if ($this->windows_version > '6.1.0000') {
+            if (version_compare($this->windows_version,'6.1.0000')>0) {
                 $canonname = preg_replace('/[^A-Za-z0-9- ]/', '_', $net->Name);
+                $canonname2 = str_replace('(','[',$net->Name);
```

Comment:
I'd prefer to not have `str_replace()` twice. You can combine those two replaces into a single function call using either of the following. Please pick one of those :)
```php
$canonname2 = strtr($canonname, ['(' => '[', ')' => ']']);
```

or this:

```php
$canonname2 = str_replace(['(', ')'], ['[', ']'], $cannonname);
```